ideascube/serveradmin/catalog.py

```diff
@@ -18,6 +18,14 @@
 
 from .systemd import Manager as SystemManager, NoSuchUnit
 
+# We do not use functool.partial cause we want to mock stderr for unittest
+# If we use partial we keep a ref to the original sys.stderr and output is not
+# captured.
+def printerr(*args, **kwords):
```

Comment:
Because ! :)
But agreed, kwargs is better.
+1 also for moving printerr into util.py.